canonicalwebteam/http/__init__.py

```diff
@@ -0,0 +1,105 @@
+# Core packages
```

Comment:
Similarly, to my previous comment, I'm aware that some people feel comments should be removed except where absolutely needed (in favour of self-documenting code), I personally think it's usually better to err on the side of more comments.

People reading our code are usually less knowledgeable than we would think they are, and I'd rather not only write code for "experience Python devs" or "experience Flask devs" or whatever.

So in this case, unless you feel very strongly about it, I'd rather keep the comment.

This sort of leads to a principle I have that I'd like on the whole to keep our criticism of pull-requests to things with significant impact, so we don't spend a lot of time bike-shedding, and examples of such things to me include discussions about nuanced code style, strict opinions about commit messages, etc.

However this is just my opinions and If you want to have a wider discussion about this, and the previous item, please go ahead and suggest a PR on [the practices repo](https://github.com/canonical-webteam/practices) and it will start a discussion with the whole team and we can come to a consensus.

Reply:
Good point, that does sound interesting. I'll have a search, or feel free to link me to any good ones you know.

Reply:
@tbille was having fun with an emacs extension that does this